.gitignore

```diff
@@ -106,4 +106,3 @@ venv.bak/
 
 static/
 .idea/*
-*/migrations/*
```

Comment:
😬 why were we ignoring these in the past?

Reply:
idfk, I thought I removed that line a while ago